spec/jobs/refresh_mods_job_spec.rb

```diff
@@ -8,15 +8,19 @@
   let(:bulk_action) { create(:bulk_action) }
   let(:user) { bulk_action.user }
 
-  let(:catkeys) do
+  let(:catalog_record_ids) do
     ["123"]
   end
 
+  # NOTE: the `catkeys` arg in the cocina-models factories is currently the only
+  #       possibility for catalog record IDs, and those map to Symphony catalog
+  #       links. The next release (after 0.89.0) should contain a change
+  #       allowing both the `catkeys` arg and the `folio_instance_hrids` arg
```

Comment:
The work's already been done in cocina-models. I think the ticket we need may be Argo ticket to update this code once the next cocina-models release is cut. But we don't know when that is, so I'm open to suggestions for how to sequence/plan that. Also happy to have this sit here since it's not super important.